### PR TITLE
micro:bit Image

### PR DESCRIPTION
**Pull request by xnkevinnguyen**
# Description:
This pr is dependant of this pr https://github.com/microsoft/vscode-python-devicesimulator/pull/183
The microbit image is rendered under the micro:bit tab.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/view/components/cpx/CpxImage.tsx [hunk 1]
```diff
@@ -20,30 +20,34 @@ interface IProps {
...
-
-    if (svgElement) {
-        if (firstTime) {
-            initSvgStyle(svgElement, props.brightness);
-            setupButtons(props);
-            setupPins(props);
-            setupKeyPresses(props.onKeyEvent);
-            setupSwitch(props);
-            firstTime = false;
+//
```
Comment (nasadigital) [inline src/view/components/cpx/CpxImage.tsx]: Perhaps remove this or add a comment?

**Review by nasadigital — APPROVED** [review]
LGTM!
I'd have @sagarmanchanda take a look before merging, though.

File: src/view/components/cpx/CpxImage.tsx (hunk 1 above)
Comment (xnkevinnguyen) [reply src/view/components/cpx/CpxImage.tsx]: thanks david, i'll remove this

File: src/view/styles/Simulator.css [hunk 2]
```diff
@@ -63,12 +64,9 @@
...
-.simulator {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    max-width: 700px;
-    max-height: 700px;
-    margin-left: auto;
-    margin-right: auto;
+.microbit-container {
+    max-width: 350px;
```
Comment (sagarmanchanda) [inline src/view/styles/Simulator.css:68]: Why is this not also 100% as cpx?

**Review by sagarmanchanda — APPROVED** [review]

File: src/view/styles/Simulator.css (hunk 2 above)
Comment (xnkevinnguyen) [reply src/view/styles/Simulator.css:68]: There is already some max sizing in the default cpx styling, but not on micro-bit

**Pull request merged**